analyticsclient/tests/test_course.py

```diff
@@ -149,6 +149,23 @@ def test_problems(self):
         self.assertEqual(body, self.course.problems())
 
     @httpretty.activate
+    def test_typology(self):
+
+        body = [
+            {
+                "chapter_id": "week2",
+                "video_type": 2,
+                "problem_type": 1,
```

Comment:
@bradenmacdonald This would be a good place to use the constants you added to `constants/typology.py`:

```python
from analyticsclient.constants.typology import ALL, SOME
# ...
                "video_type": ALL,
                "problem_type": SOME,
```